source/user/scf.rst

```diff
@@ -175,7 +175,12 @@ the previous or other calculations::
 
 ``/path/to/chkfile`` can be found in the output in the calculation (if
 ``mol.verbose >= 4``, the filename of the chkfile will be dumped in
-the output).  By setting :attr:`chkfile` and :attr:`init_guess`, the
+the output).  If there is a chance a later restart might be required,
+it is recommended to set :attr:`chkfile` explicitly in the
+earlier calculation as the chkfile might otherwise be deleted upon
+successful completion, see comments in
+:source:`examples/scf/14-restart.py`.
+By setting :attr:`chkfile` and :attr:`init_guess`, the
```

Comment:
I would suggest:

If the results of the calculation are needed at a later stage (e.g. for an eventual restart or use as an initial guess for a larger calculation), the :attr:`chkfile` attribute should be set explicitly as the chkfile might otherwise be deleted upon successful completion of the calculation, see comments in :source:`examples/scf/14-restart.py`.

Reply:
Sounds good, I have replaced it with your suggestion.